clock_exercise/ClockButton.qml

```diff
@@ -3,8 +3,11 @@ import QtQuick 2.0
 Rectangle {
     id: button
 
+    property int timeOfDissolve: 500
```

Comment:
Vedo che più in basso in questo file stai usando questa property con `duration: timeOfDissolve` invece che con `duration: button.timeOfDissolve`. Fai attenzione perchè non sempre succede quello che ti aspetti, quando sei dentro componenti diverse cerca di specificare sempre l'ID di riferimento.

Oltre a questo, aggiungo un paio di osservazioni:
* se questa property è previsto che sia costante, possiamo marchiarla readonly, così il lettore capisce che non viene modificata da nessuna parte
* se vogliamo usare una costante per le duration da più parti, possiamo valutare se esporre una costante direttamente da C++, così sarà uguale per tutti e non dovremo aggiungere tante volte una stessa property in componenti diverse, almeno abbiamo anche un solo punto di modifica globale per tutte le duration; prova a valutare se ci faccia comodo impostarla così o se la cosa apporti meno benefici del previsto